### ISSUE TITLE
Generate SQL feature

### ISSUE THREAD

**Issue by gavr123456789** [Alecaddd/sequeler#339]
PCM on any record calls Popover with the ability to generate a Select, Insert, Delete, Update SQL query for this record. 
The idea is taken from DBeaver.

**Comment by Alecaddd** [comment]
indeed, this is a feature that have been discussed in other issues and it's definitely on my radar for future updates.
I got to find the time :sweat_smile:
